### PR TITLE
[DOC-3239] Release notes for v22.2.0-beta.1

### PR DESCRIPTION
**Pull request by MichaelTrestman**

### CODE REVIEW

**Comment by nickvigilante** [comment]
@MichaelTrestman Can you amend your commit and force-push? Retrying the build in Netlify did not properly sync the status to GitHub.

**Comment by MichaelTrestman** [comment]
@nickvigilante did that do what it was suppose to?

**Comment by nickvigilante** [comment]
Yes it did. Thank you!

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+- `debug zip`'s `--redact-logs` flag has been deprecated in favor of the `--redact` flag, which applies to a broader scope than just logs (but also includes logs). The new `--redact` flag will trigger the redaction of all sensitive data in debug zip bundles, except for range keys, which have been deemed necessary to keep unredacted as they are essential to support CockroachDB. The `--redact-logs` flag will still remain, but users of debug zip will be warned of its deprecation if they use it, and it will be interpreted as `--redact` instead.  Release justification: low risk, high value change necessary for upcoming compliance mandates [#88266][#88266]
+
+<h3 id="v22-2-0-beta-1-db-console-changes">DB Console changes</h3>
+
+- On the SQL Activity Session Details page the Most Recent Statement show the last active query instead of No Active Statement. [#88057][#88057]
+- Update filter labels from "App" to "Application Name" and from "Username" to "User Name" on SQL Activity pages. [#88144][#88144]
+- Transaction execution id column now exists in the blocked statements table in the active stmt details page [#88135][#88135]
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:27]:
```suggestion
- The **Transaction Execution ID** column now exists in the blocked statements table in the active stmt details page [#88135][#88135]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+<h3 id="v22-2-0-beta-1-bug-fixes">Bug fixes</h3>
+
+- This patch fixes incorrect results from queries which utilize locality-optimized search on the inverted index of a table with REGIONAL BY ROW partitioning. [#88114][#88114]
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:31]:
```suggestion
- This patch fixes incorrect results from queries which utilize locality-optimized search on the inverted index of a table with `REGIONAL BY ROW` partitioning. [#88114][#88114]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- The function `pg_catalog.pg_get_viewdef` now works properly for materialized views. [#88143][#88143]
+- Cluster and tenant restores of dropped or adding tenants would incorrectly activate those tenants during restore. [#88157][#88157]
+- The `current_setting` builtin function now properly does not result in an error when checking a custom session setting that does not exist and the `missing_ok` argument is true. [#88158][#88158]
+- `CREATE TABLE foo as (SELECT * FROM crdb_internal.check_consistency(...))` no longer panics the gateway node. [#88227][#88227]
+- Previously, CockroachDB could incorrectly not fetch rows with NULL values when reading from the unique secondary index when multiple column families are defined for the table and the index doesn't store some of the NOT NULL columns. [#88210][#88210]
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:36]:
```suggestion
- Previously, CockroachDB could incorrectly not fetch rows with `NULL` values when reading from the unique secondary index when multiple column families are defined for the table and the index doesn't store some of the `NOT NULL` columns. [#88210][#88210]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+- Fixed crashes in changefeed code when running on recent go versions. [#88231][#88231]
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:37]:
```suggestion
- Fixed crashes in changefeed code when running on recent Go versions. [#88231][#88231]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+- For statement execution insights, the value for the end time was updated to be congruent with the start time and latency. [#88276][#88276]
+- Completed stmt diagnostics bundles now persist in the UI in stmt diag bundle pages [#88286][#88286]
+- Dropping temporary tables and sequences should check privilege too. [#88305][#88305]
+- The pgwire DESCRIBE step no longer fails with an error while attempting to look up cursors declared with names containing special characters. [#88316][#88316]
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:41]:
```suggestion
- The pgwire `DESCRIBE` step no longer fails with an error while attempting to look up cursors declared with names containing special characters. [#88316][#88316]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+- Jobs pages refreshes data at an interval of 10s [#88285][#88285]
+- Fixed a crash in changefeed expressions when JSON encoding an empty projection. [#88311][#88311]
+- CockroachDB now more promptly reacts to query cancellations (e.g. due to statement timeout being exceeded) after the query spilled to disk. [#88378][#88378]
+
+<h3 id="v22-2-0-beta-1-doc-updates">Doc updates</h3>
```
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:46]:
```suggestion
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+Release Date: September 26, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-1-security-updates">Security updates</h3>
+
+- Redacted logs will now reveal pretty-printed keys, except for the index key values themselves. For example `/Table/42/1222/‹x›` will be shown instead of `‹x›` (which was shown previously). This improved redaction is available for the `/Table` keyspace for both system and application tenants. Other keyspaces such as `/Meta1`, `/Meta2`, `/Local`, etc. are not yet supported.  Release justification: low risk, high benefit observability changes [#87647][#87647]
+- The following types of data are now considered "safe" for reporting from within debug.zip:
+    - Range start/end keys, which can include data from any indexed   SQL column. - Key spans, which can include data from any indexed SQL column
```
Comment (stbof) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:11]: This looks like two bullets joined together.

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+- Previously, CockroachDB could incorrectly not fetch rows with `NULL` values when reading from the unique secondary index when multiple column families are defined for the table and the index doesn't store some of the `NOT NULL` columns. [#88210][#88210]
+- Fixed crashes in changefeed code when running on recent Go versions. [#88231][#88231]
+- For statement execution insights, the value for the end time was updated to be congruent with the start time and latency. [#88276][#88276]
+- Completed stmt diagnostics bundles now persist in the UI in stmt diag bundle pages [#88286][#88286]
+- Dropping temporary tables and sequences should check privilege too. [#88305][#88305]
+- The pgwire `DESCRIBE` step no longer fails with an error while attempting to look up cursors declared with names containing special characters. [#88316][#88316]
+- Jobs pages refreshes data at an interval of 10s [#88285][#88285]
+- Fixed a crash in changefeed expressions when JSON encoding an empty projection. [#88311][#88311]
+- CockroachDB now more promptly reacts to query cancellations (e.g. due to statement timeout being exceeded) after the query spilled to disk. [#88378][#88378]
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:48]:
```suggestion
- CockroachDB now reacts more quickly to query cancellations (such as when a statement timeout is exceeded) after a query is spilled to disk. [#88378][#88378]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+
+{% comment %}Docs team: Please add these manually.{% endcomment %}
+
+<h3 id="v22-2-0-beta-1-contributors">Contributors</h3>
+
+This release includes 54 merged PRs by 30 authors.
+We would like to thank the following contributors from the CockroachDB community:
+
+- Mark Sirek (first-time contributor, CockroachDB team member)
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:58]: Check the [authors file](https://github.com/cockroachdb/cockroach/blob/master/AUTHORS) and remove any CRL employees from this list. If you remove them all, also remove line 56.

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
+## v22.2.0-beta.1
+
+Release Date: September 26, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-1-security-updates">Security updates</h3>
+
+- Redacted logs will now reveal pretty-printed keys, except for the index key values themselves. For example `/Table/42/1222/‹x›` will be shown instead of `‹x›` (which was shown previously). This improved redaction is available for the `/Table` keyspace for both system and application tenants. Other keyspaces such as `/Meta1`, `/Meta2`, `/Local`, etc. are not yet supported.  Release justification: low risk, high benefit observability changes [#87647][#87647]
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply _includes/releases/v22.2/v22.2.0-beta.1.md:9]: I added the vocab but the rest of the suggestion is still valid. :)

**Review by mdlinville — COMMENTED** [review]
I didn't do all of these, but each bullet should end in a period.

File: _includes/releases/v22.2/v22.2.0-beta.1.md (hunk 7 above)
Comment (MichaelTrestman) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:11]:
```suggestion
    - Range start/end keys, which can include data from any indexed   SQL column.
    - Key spans, which can include data from any indexed SQL column
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+- On the SQL Activity Session Details page the Most Recent Statement show the last active query instead of No Active Statement. [#88057][#88057]
+- Update filter labels from "App" to "Application Name" and from "Username" to "User Name" on SQL Activity pages. [#88144][#88144]
+- The **Transaction Execution ID** column now exists in the blocked statements table in the active stmt details page [#88135][#88135]
+
+<h3 id="v22-2-0-beta-1-bug-fixes">Bug fixes</h3>
+
+- This patch fixes incorrect results from queries which utilize locality-optimized search on the inverted index of a table with `REGIONAL BY ROW` partitioning. [#88114][#88114]
+- The function `pg_catalog.pg_get_viewdef` now works properly for materialized views. [#88143][#88143]
+- Cluster and tenant restores of dropped or adding tenants would incorrectly activate those tenants during restore. [#88157][#88157]
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:38]:
```suggestion
- When restoring a cluster or tenant from a backup, only tenants that were activated during the backup are activated by the restore. [#88157][#88157]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+
+{% comment %}Docs team: Please add these manually.{% endcomment %}
+
+<h3 id="v22-2-0-beta-1-contributors">Contributors</h3>
+
+This release includes 54 merged PRs by 30 authors.
+We would like to thank the following contributors from the CockroachDB community:
+
+- Mark Sirek (first-time contributor, CockroachDB team member)
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:59]:
```suggestion
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+- Rima Deodhar (first-time contributor, CockroachDB team member)
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:60]:
```suggestion
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+- Fixed a crash in changefeed expressions when JSON encoding an empty projection. [#88311][#88311]
+- CockroachDB now more promptly reacts to query cancellations (e.g. due to statement timeout being exceeded) after the query spilled to disk. [#88378][#88378]
+
+
+{% comment %}Docs team: Please add these manually.{% endcomment %}
+
+<h3 id="v22-2-0-beta-1-contributors">Contributors</h3>
+
+This release includes 54 merged PRs by 30 authors.
+We would like to thank the following contributors from the CockroachDB community:
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:57]:
```suggestion
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+
+<h3 id="v22-2-0-beta-1-security-updates">Security updates</h3>
+
+- Redacted logs will now reveal pretty-printed keys, except for the index key values themselves. For example `/Table/42/1222/‹x›` will be shown instead of `‹x›` (which was shown previously). This improved redaction is available for the `/Table` keyspace for both system and application tenants. Other keyspaces such as `/Meta1`, `/Meta2`, `/Local`, etc. are not yet supported.  Release justification: low risk, high benefit observability changes [#87647][#87647]
+- The following types of data are now considered "safe" for reporting from within debug.zip:
+    - Range start/end keys, which can include data from any indexed   SQL column.
+    - Key spans, which can include data from any indexed SQL column
+    - Usernames and role name
+    - SQL object names (including DB, schema, table, sequence, view,   type, and UDF names)
+    Release justification: high value observability changes necessary to meet our upcoming compliance mandates. [#88266][#88266]
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:15]:
```suggestion
[#88266][#88266]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+- The following types of data are now considered "safe" for reporting from within debug.zip:
+    - Range start/end keys, which can include data from any indexed   SQL column.
+    - Key spans, which can include data from any indexed SQL column
+    - Usernames and role names
+    - SQL object names (including database, schema, table, sequence, view, type, and UDF names)
+[#88266][#88266]
+
+<h3 id="v22-2-0-beta-1-sql-language-changes">SQL language changes</h3>
+
+- Postgres-wire-level `PREPARE` statements now support the case where the number of the type hints is greater than the number of placeholders in the given query. [#88146][#88146]
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:19]:
```suggestion
- PostgresSQL wire-level `PREPARE` statements now support the case where the number of the type hints is greater than the number of placeholders in the given query. [#88146][#88146]
```

File: _includes/releases/v22.2/v22.2.0-beta.1.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+- On the SQL Activity Session Details page, the Most Recent Statement section shows the last active query instead of "No Active Statement". [#88057][#88057]
+- On the SQL Activity page, the **App** filter label has been renamed to **Application Name**, and the **Username** label has been renamed to **User Name**. [#88144][#88144]
+- The **Transaction Execution ID** column is now shown in the **Blocked Statements** table on the Active Statement Details page. [#88135][#88135]
+
+<h3 id="v22-2-0-beta-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a problem that could cause incorrect results from queries that use locality-optimized search on the inverted index of a table with `REGIONAL BY ROW` partitioning. [#88114][#88114]
+- The `pg_catalog.pg_get_viewdef` function now works properly for materialized views. [#88143][#88143]
+- When restoring a cluster or tenant from a backup, only tenants that were activated during the backup are activated during the restore. [#88157][#88157]
+- Fixed a problem where checking a custom sessionsetting with the `current_setting` built-in function resulted in an error if the setting was not set and the `missing_ok` argument was true. [#88158][#88158]
```
Comment (mdlinville) [inline _includes/releases/v22.2/v22.2.0-beta.1.md:39]:
```suggestion
- Fixed a problem where checking a custom session setting with the `current_setting` built-in function resulted in an error if the setting was not set and the `missing_ok` argument was true. [#88158][#88158]
```

**Review by mdlinville — APPROVED** [review]

**Pull request merged**